modules/openapi-generator/src/main/resources/typescript/model/ObjectSerializer.mustache

```diff
@@ -162,4 +168,73 @@ export class ObjectSerializer {
             return instance;
         }
     }
+
+
+    /**
+     * Normalize media type
+     *
+     * We currently do not handle any media types attributes, i.e. anything
+     * after a semicolon. All content is assumed to be UTF-8 compatible.
+     */
+    public static normalizeMediaType(mediaType: string | undefined): string | undefined {
+        if (mediaType === undefined) {
+            return undefined;
+        }
+        return mediaType.split(";")[0].trim().toLowerCase();
```

Comment:
Is `;` required at the end of a mediaType/http header field if no additional types follow?

Reply:
`;` separates the parameters from the main media type. The function itself only handles one media type. So multiples would have to be split beforehand.
If there is no `;` that is ok, because then `.split` returns an array with one element that is the whole media type.